### ISSUE TITLE
Preserve Classes folder structure in Cocoapods

### ISSUE THREAD

**Issue by RobertWalsh** [apigee/IgniteEngine-iOS#127]
Since moving to only having the Starter Kit project and using Cocoapods exclusively, we need to configure the podspec to at the very least preserve the Classes folder hierarchy.

Developing and going through the classes folder as it is currently is tremendously tedious.  This can be done with the use of subspecs.  For examples on how to do that check out these:

 https://github.com/AFNetworking/AFNetworking/blob/master/AFNetworking.podspec

https://github.com/CocoaLumberjack/CocoaLumberjack/blob/master/CocoaLumberjack.podspec

Also should check out this http://www.rubydoc.info/gems/cocoapods/Pod/Project#add_file_reference-instance_method

**Comment by r3mus** [comment]
We do this already with `spec.preserve_paths = 'Classes/**'`, however CocoaPods only permits this if you set it up as a development pod. Clone the source somewhere, then in your podfile, change it to:

    pod 'IgniteEngine', :path => '../_IXSource'

**Comment by RobertWalsh** [comment]
No.  Look at AFNetworking like I said.  See the subfolders? See the structure? I didn't have to point that to anything in my podfile.  This is how it should/needs to look like.  


![screen shot 2015-05-24 at 11 54 33 pm](https://cloud.githubusercontent.com/assets/1379821/7792024/46253c2c-0270-11e5-8b7e-84ab0f866b2f.png)

**Comment by r3mus** [comment]
Misunderstood the objective of what you were trying to do. We can certainly do this, but if you're developing, using the local :path param will be much more effective as it symlinks directly to your cloned repo.

**Comment by r3mus** [comment]
Implemented in https://github.com/r3mus/IgniteEngine-iOS/commit/9145255e53451b4634c30c1d337845aaba3d935b
